charts/celestia-local/templates/ingress.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
Added ingress which makes it possible to communicate directly with deployed Celestia app and bridge node from local machine.